File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/ResponseModelClass.java

package restaurant.sa.com.sarestaurant.appview.restaurant.model;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ResponseModelClass {

    @SerializedName("html_attributions")
    private List<Object> htmlAttributions;
    @SerializedName("next_page_token")
    private String nextPageToken;
    @Expose
    private List<Result> results;
    @Expose
    private String status;

    public List<Object> getHtmlAttributions() {
        return htmlAttributions;
    }

    public void setHtmlAttributions(List<Object> htmlAttributions) {
        this.htmlAttributions = htmlAttributions;
    }

    public String getNextPageToken() {
        return nextPageToken;
    }

    public void setNextPageToken(String nextPageToken) {
        this.nextPageToken = nextPageToken;
    }

    public List<Result> getResults() {
        return results;
    }

    public void setResults(List<Result> results) {
        this.results = results;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

}

File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/ResponseModelClass.kt
package restaurant.sa.com.sarestaurant.appview.restaurant.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ResponseModelClass {

    @SerializedName("html_attributions")
    var htmlAttributions: List<Any>? = null
    @SerializedName("next_page_token")
    var nextPageToken: String? = null
    @Expose
    var results: List<Result>? = null
    @Expose
    var status: String? = null

}
